lib/stages/visAccumulate.cpp

```diff
@@ -304,6 +305,13 @@ void visAccumulate::main_thread() {
 
         // Start and end times of this frame
         timespec t_s = ((chimeMetadata*)in_buf->metadata[in_frame_id]->metadata)->gps_time;
+        // If GPS time is not set, fall back to the system's best guess at the time.
+        if (t_s.tv_sec == 0) {
+            TIMEVAL_TO_TIMESPEC(
+                &((chimeMetadata*)in_buf->metadata[in_frame_id]->metadata)->first_packet_recv_time,
+                &t_s);
+            WARN("GPS time not set, using much less accurate system time instead.");
```

Comment:
If anyone expects that they might want to run in this state, this is going to be a very verbose message. Maybe only warn the first time (or when `gps_time_enabled()` is evaluated if doing the above suggestion).